lntest/btcd.go

```diff
@@ -81,6 +81,7 @@ func NewBackend(miner string, netParams *chaincfg.Params) (
 		"--debuglevel=debug",
 		"--logdir=" + logDir,
 		"--connect=" + miner,
+		"--nowinservice",
```

Comment:
We technically don't need to _always_ specify this flag, but it doesn't hurt at all I guess.